hl/tools/h5watch/testh5watch.sh.in

```diff
@@ -13,16 +13,12 @@
 #
 # Tests for the h5watch tool
 #
-bindir=@bindir@
-
-# If the bindir directory is not set just use current (.).
-if test -z "$bindir"; then
-   bindir=.
-fi
+top_builddir=@abs_top_builddir@
+utilstestdir=$top_builddir/@utilstestdir@
```

Comment:
utilstestdir=$top_builddir/@H5_UTILS_TEST_BUILDDIR@

Also, it'd probably be easier to read if the variable name had some _ spacers. e.g., utils_test_dir